File: plugins/OrchidJavadoc/src/main/java/com/eden/orchid/javadoc/resources/PackageDocResource.java
package com.eden.orchid.javadoc.resources;

import com.eden.orchid.api.OrchidContext;
import com.sun.javadoc.PackageDoc;

public class PackageDocResource extends BaseJavadocResource {

    public PackageDocResource(OrchidContext context, PackageDoc packageDoc) {
        super(context, packageDoc.name(), packageDoc);
    }
}

File: plugins/OrchidJavadoc/src/main/kotlin/com/eden/orchid/javadoc/resources/PackageDocResource.kt
package com.eden.orchid.javadoc.resources

import com.copperleaf.javadoc.json.models.JavaPackageDoc
import com.eden.orchid.api.OrchidContext

class PackageDocResource(
        context: OrchidContext,
        packageDoc: JavaPackageDoc
) : BaseJavadocResource(context, packageDoc.name, packageDoc)
